src/main/scala/org/broadinstitute/dsde/workbench/avram/db/CollectionComponent.scala

```diff
@@ -0,0 +1,65 @@
+package org.broadinstitute.dsde.workbench.avram.db
+
+import java.sql.Timestamp
+import java.time.Instant
+
+import org.broadinstitute.dsde.workbench.avram.api.Collection
+import AvramPostgresProfile.api._
+
+import scala.concurrent.ExecutionContext
+
+case class CollectionRecord(id: Long,
+                            name: String,
+                            createdBy: String,
+                            samResource: String,
+                            createdTimestamp: Timestamp,
```

Comment:
I'd prefer to keep `java.sql` classes out of our domain model. Also, `java.sql.Timestamp` has issues (e.g. it's a `java.util.Date` but doesn't conform to the equals/hashCode contract... and appears to use precision if used as the key of a hash map).

Maybe we can convert it to a `java.time.Instant`?